lib/FieldLocation/FieldLocationFinalContainer.js

```diff
@@ -0,0 +1,82 @@
+import React, {
+  useState,
+  useCallback,
+  useEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+import {
+  find,
+  uniqBy,
+} from 'lodash';
+
+import {
+  Loading,
+} from '@folio/stripes/components';
+
+import {
+  getLocationOptions,
+} from '../utils';
+import FieldLocationFinal from './FieldLocationFinal';
+
+const FieldLocationFinalContainer = ({
+  isDisabled,
+  labelId,
+  locationLookupLabel,
+  locationsForDict,
+  name,
+  onChange,
+  prepopulatedLocationsIds,
+  required,
+}) => {
+  const [locationsForSelect, setLocationsForSelect] = useState();
+
+  useEffect(() => {
+    const locations = prepopulatedLocationsIds?.map(id => find(locationsForDict, { id })) || [];
+
+    setLocationsForSelect(getLocationOptions(locations));
+  },
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  []);
+
+  const selectLocationFromPlugin = useCallback(
+    (location) => {
+      setLocationsForSelect(prev => uniqBy([...prev, ...getLocationOptions([location])], 'value'));
+
+      onChange(location);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
could you please read regarding this line? You're using it almost everywhere, but this is not the purpose of this line
https://github.com/facebook/create-react-app/issues/6880#issuecomment-485912528
https://stackoverflow.com/a/58959607

Reply:
fixed